integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java

```diff
@@ -7,15 +7,13 @@
 
   import org.json.JSONObject;
   import org.junit.Test;
-  import org.opensearch.sql.legacy.SQLIntegTestCase;
 
   import java.io.IOException;
 
   import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_DATE;
   import static org.opensearch.sql.util.MatcherUtils.rows;
   import static org.opensearch.sql.util.MatcherUtils.schema;
-  import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
-  import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+    import static org.opensearch.sql.util.MatcherUtils.verifySchema;
```

Comment:
Do you know why this import is indented?

Reply:
Not sure. Fixed here: [afd157b](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/commits/afd157b195ad8301577d643b08d019103afecd9e)
Merge state
Add more commits by pushing to the [dev/sl_removeUnusedImports](https://github.com/Bit-Quill/opensearch-project-sql/tree/dev/sl_removeUnusedImports) branch on [Bit-Quill/opensearch-project-sql](https://github.com/Bit-Quill/opensearch-project-sql).

Changes approved
3 approving reviews 
3 approvals
3 pending reviewers
Some checks were not successful
4 failing and 16 successful checks
@github-actions
SQL Java CI / build (ubuntu-latest, 11) (pull_request) Failing after 27m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216639547?pr=305)
@github-actions
SQL Java CI / build (ubuntu-latest, 11) (push) Failing after 27m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216274919?pr=305)
@github-actions
CodeQL / CodeQL-Scan (java) (push) Successful in 6m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621941/job/15216274923?pr=305)
@github-actions
Developer Certificate of Origin Check / check (pull_request) Successful in 5s
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736615/job/15216639543?pr=305)
@github-actions
Link Checker / linkchecker (pull_request) Successful in 10s
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736616/job/15216639545?pr=305)
@github-actions
Release Drafter / Update draft release notes (pull_request) Successful in 1m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736618/job/15216639528?pr=305)
@github-actions
SQL Java CI / build (windows-latest, 11, -x doctest -PbuildPlatform=windows) (pull_request) Successful in 21m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216639743?pr=305)
@github-actions
SQL Java CI / build (windows-latest, 11, -x doctest -PbuildPlatform=windows) (push) Successful in 21m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216275072?pr=305)
@github-actions
SQL Java CI / build (macos-latest, 11) (pull_request) Successful in 25m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216639932?pr=305)
@github-actions
SQL Java CI / build (macos-latest, 11) (push) Successful in 23m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216275232?pr=305)
@github-actions
SQL Java CI / build (ubuntu-latest, 17) (pull_request) Failing after 29m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216640113?pr=305)
@github-actions
SQL Java CI / build (ubuntu-latest, 17) (push) Failing after 33m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216275408?pr=305)
@github-actions
SQL Java CI / build (windows-latest, 17, -x doctest -PbuildPlatform=windows) (pull_request) Successful in 16m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216640293?pr=305)
@github-actions
SQL Java CI / build (windows-latest, 17, -x doctest -PbuildPlatform=windows) (push) Successful in 17m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216275562?pr=305)
@github-actions
SQL Java CI / build (macos-latest, 17) (pull_request) Successful in 25m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615736617/job/15216640469?pr=305)
@github-actions
SQL Java CI / build (macos-latest, 17) (push) Successful in 22m
[Details](https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/5615621934/job/15216275720?pr=305)
@github-code-scanning
Code scanning results / CodeQL Successful in 1s — No new alerts in code changed by this pull request
[Details](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/checks?check_run_id=15216638072)
@dco
DCO Successful in 1s — DCO
[Details](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/checks?check_run_id=15216638558)
@codecov
codecov/patch — Coverage not affected when comparing dade5d5...428153d
[Details](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/checks?check_run_id=15217625923)
@codecov
codecov/project — 97.39% (target 99.00%), passed because coverage was not affected by patch
[Details](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/checks?check_run_id=15217625943)
This branch has no conflicts with the base branch
Merging can be performed automatically.
 You can also [open this in GitHub Desktop](https://desktop.github.com/) or view .
@MitchellGale

 
Leave a comment
No file chosen
Attach files by dragging & dropping, selecting or pasting them.
Remember, contributions to this repository should follow its [contributing guidelines](https://github.com/Bit-Quill/opensearch-project-sql/blob/6c3744ec81931567d50f47075cd45b3e8906997c/CONTRIBUTING.md), [security policy](https://github.com/Bit-Quill/opensearch-project-sql/blob/6c3744ec81931567d50f47075cd45b3e8906997c/SECURITY.md), and [code of conduct](https://github.com/Bit-Quill/opensearch-project-sql/blob/6c3744ec81931567d50f47075cd45b3e8906997c/CODE_OF_CONDUCT.md).
 ProTip! Add comments to specific lines under [Files changed](https://github.com/Bit-Quill/opensearch-project-sql/pull/305/files).
Reviewers
@[forestmvey](https://github.com/forestmvey)
forestmvey
@[Yury-Fridlyand](https://github.com/Yury-Fridlyand)
Yury-Fridlyand
@[matthewryanwells](https://github.com/matthewryanwells)
matthewryanwells
@[MaxKsyunz](https://github.com/MaxKsyunz)
MaxKsyunz
@[acarbonetto](https://github.com/acarbonetto)
acarbonetto
@[GumpacG](https://github.com/GumpacG)
GumpacG
Still in progress?
Assignees
No one—
Labels
None yet
Projects
None yet
Milestone
No milestone
Notifications
Customize
You’re receiving notifications because you authored the thread.
4 participants
@MitchellGale
@forestmvey
@Yury-Fridlyand
@matthewryanwells